.dep-versions

```diff
@@ -5,4 +5,4 @@ llvm=cd9a641613eddf25d4b25eaa96b2c393d401d42c
 enzyme=1beb98b51442d50652eaa3ffb9574f4720d611f1
 
 # Always remove custom PL/LQ versions before release.
-pennylane=7f97ea735b3c43b43a0ffe34040bffaa49603268
\ No newline at end of file
+pennylane=4eb803abf19c2438b8d4d208525a727c294b2bbd
```

Comment:
This is a reminder for later to double check which hash is newer once changes from main are incorporated.